docs/content/latest/secure/audit-logging/audit-logging-ysql.md

```diff
@@ -168,85 +174,77 @@ There is no default.
   </tr>
 </table>
 
-
 ## Example
 
 Use these steps to configure audit logging in a YugabyteDB cluster with bare minimum configurations.
 
 
 ### 1. Enable audit logging
 
-Start the YugabyteDB Cluster with the following Audit logging Configuration.
-
-    ```
-    --ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice,pgaudit.log_client=ON"
-    ```
-
+Start the YugabyteDB Cluster with the following Audit logging Configuration:
 
+```shell
+--ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice,pgaudit.log_client=ON"
+```
 
-    Alternatively, go to a YSQL shell and execute the following commands.
+Alternatively, open the YSQL shell and execute the following commands:
 
 
-    ```
-    SET pgaudit.log='DDL'; 
-    SET pgaudit.log_client=ON;
-    SET pgaudit.log_level=notice;
-    ```
+```shell
+SET pgaudit.log='DDL'; 
+SET pgaudit.log_client=ON;
+SET pgaudit.log_level=notice;
+```
 
 
 ### 2. Load `pgAudit` extension
```

Comment:
pgaudit?

Reply:
It's pgAudit if not in code

Reply:
OK wasn't sure - Later in text there is "To enable the pgaudit extension" and then just after "Finally, create the pgAudit extension as follows:"